cookbooks/bcpc-hadoop/attributes/disks.rb

```diff
@@ -0,0 +1,10 @@
+# enumerate available disks and how they will be used
+default[:bcpc][:hadoop][:disks] = {
+  :available_disks => ["sdb" , "sdc", "sdd", "sde"],
+  # keep at least that many disks for the :disk_exclude_role
+  :role_min_disk => 2,
+  # we are reserving disks for the following
+  :reservation_requests => ["graphite_disk"],
+  # reservations will be saved here
+  :disk_exclude_roles => ["BCPC-Hadoop-Head"]
```

Comment:
Why is the variable named disk_exclude_roles?  It sounds like the roles listed will be excluded from the reservation request.  It would be better to have a name like remaining_disks_roles.